provider/baked/src/export.rs

```diff
@@ -633,13 +651,17 @@ impl DataExporter for BakedExporter {
                         }
                     }
                 },
-                quote! {
-                    #maybe_msrv
-                    impl icu_provider::DryDataProvider<#marker_bake> for $provider {
-                        fn dry_load(&self, req: icu_provider::DataRequest) -> Result<icu_provider::DataResponseMetadata, icu_provider::DataError> {
-                            icu_provider::DataProvider::<#marker_bake>::load(self, req).map(|r| r.metadata)
+                if deduplication_mode == DeduplicationStrategy::RetainBaseLanguages {
```

Comment:
Nit: It should also be output if deduplication is None

Reply:
Done. Should we just call it `DRY` then? `MAYBE_DRY`? `TRY_DRY`?

Reply:
Hmm, I'm okay with `DRY` if the default behavior is to simply not export it if maximal deduplication.